jwst/ramp_fitting/ramp_fit.py

```diff
@@ -443,10 +444,19 @@ def ols_ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
                                        group_time, max_seg )
 
             segs_4[num_int, :, rlo:rhi, :] = segs_beg_3
+
+            # Suppress harmless arithmetic warnings for now
+            warnings.filterwarnings("ignore", ".*invalid value.*",
+                RuntimeWarning)
+            warnings.filterwarnings("ignore", ".*divide by zero.*",
+                RuntimeWarning)
```

Comment:
Another way to do this would be to use `numpy.errstate` context manager.  For example:
```
with np.errstate(invalid='ignore', divide='ignore'):
    np.sqrt(-1)
    np.array([1, 1]) / np.array([1, 0])
```
Once you exit the block, the error state returns to the default.

Reply:
I.e. doing it this way will ask `numpy` to not even emit the warnings.

Reply:
I had considered that but decided I preferred to avoid the extra indentation.